docs: init (#911)
revanced/revanced-manager: docs/3_troubleshooting.md

# 🛟 Troubleshooting

In case you encounter any issues while using ReVanced Manager, please refer to this page for possible solutions.

- 💥 App not installed as package conflicts with an existing package

>   An existing installation of the application you're trying to patch is conflicting with the patched application. Uninstall the existing application before installing the patched application.

Author: This correction is not necessarily correct. It may conflict with an existing installation, not necessary patched.

> - ❗️Error code `135`, `139` or `1` when patching the application

Reviewer: ```suggestion
- ❗️Error code `135`, `139`, or `1` when patching the application
```
oxford comma good

  Your device is not supported. Refer to the [Prerequisites](0_prerequisites.md) page for supported devices.

>   Alternatively you can try to patch the application using [ReVanced CLI](https://github.com/revanced/revanced-cli) to patch the application.

Reviewer: ```suggestion
  Alternatively, you can use [ReVanced CLI](https://github.com/revanced/revanced-cli) to patch the application.
```

- 🚫 Non-root install is not possible with the current patches selection

>   Select the **Default** chip when chosing patches.

Reviewer: Oh and also
```suggestion
  Select the **Default** chip when choosing patches.
```

- 🚨 Patched application crashes on launch

>   Select the **Default** chip when chosing patches.

Reviewer: ```suggestion
  Select the **Default** chip when choosing patches.
```
